<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PeriodEdit</class>
 <widget class="QDialog" name="PeriodEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>215</width>
    <height>137</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PeriodEdit</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QComboBox" name="cb_type">
     <item>
      <property name="text">
       <string>Year</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Month</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Week</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Day</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Custom</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,1">
     <item>
      <widget class="QDateEdit" name="de_begin">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>â€“</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="de_end">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cb_type</tabstop>
  <tabstop>de_begin</tabstop>
  <tabstop>de_end</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PeriodEdit</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>156</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PeriodEdit</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>146</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
